extensions/Push/lib/Connector/Webhook.pm

```diff
@@ -155,19 +157,13 @@ sub send {
 # Private methods
 
 sub _user_agent {
-  my ($self) = @_;
-
-  my $ua = LWP::UserAgent->new(agent => 'Bugzilla');
-  $ua->timeout(10);
-  $ua->protocols_allowed(['http', 'https']);
-
-  if (my $proxy_url = Bugzilla->params->{proxy_url}) {
-    $ua->proxy(['http', 'https'], $proxy_url);
+  my $ua = Mojo::UserAgent->new(request_timeout => 30);
```

Comment:
Lets also set `connect_timeout` to a low value here (eg. 5 seconds) to avoid waiting up to 30 seconds for the initial connection to a broken server/route.